### PR TITLE
Feature/inba 957 grommet checkbox removal

### PR DESCRIPTION
**Pull request by eawest2**
#### What does this PR do?

Removes the Grommet Checkbox component in preparation for removing Grommet from Indaba completely. Replaces the two instances of Grommet checkboxes with custom checkbox components written in JSX.

#### Related JIRA tickets:

INBA-957

#### How should this be manually tested?

1) Checkout the repo and start up your indaba services and indaba-client.
2) Sign into a project management user
3) Navigate to the user's profile page at /profile, and check that their profile contains a non-functional checkbox under the heading "Is active?"
4) Navigate to any project where a user is assigned to a stage, and that stage has not yet been completed. 
5) Click on the three ellipses next to the user's name in the task assignment.
6) Once there, you should see two radio buttons and a checkbox. Click on the checkbox and check the console to make sure it's modifying the state, and make sure that the checkbox turns black when you hover over it. 

#### Screenshots (if appropriate):
capture

### CODE REVIEW

**Comment by morleegan** [comment]
@eawest2  @rileyorr - do we also want the checkbox to grey out with everything else? 
screen shot 2019-01-09 at 12 23 21 pm

**Comment by eawest2** [comment]
Good catch on the grey out Morgan, I'll check that styling change and see what I can come up with.

**Comment by eawest2** [comment]
@morleegan @rileyorr Final updates made. 

As for the styling and color of the checkbox relative to the radio buttons, I agree we should keep it consistent, but I think that may be outside the scope of this PR.

Please just take a quick look at the new changes, and let me know if they look good, and I'll merge.

**Comment by rileyorr** [comment]
@eawest2 are you able to grey out the checkbox on the user profile, please? Still looks like the user can click it.
screen shot 2019-01-10 at 10 20 41 am

LGTM otherwise!!

**Comment by eawest2** [comment]
@rileyorr Change made, please re-test. 

Thanks again guys.

**Comment by eawest2** [comment]
Duplicate <form> tag removed. Also fixed some linting errors that are showing up after I pulled the latest from develop.

File: src/views/Profile/components/ProfileCheckBox.js [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
         return (
-            <CheckBox
-                checked={typeof this.props.input.value === 'boolean'
-                    ? this.props.input.value : false }
-                value={this.props.input.value}
-                onChange={this.props.input.onChange}
-                disabled={true}
-                className='update-profile-form__checkbox' />
+            <div>
+                <form>
```
[the remark this answers is not in the document]
Comment (eawest2) [reply src/views/Profile/components/ProfileCheckBox.js]: Duplicate form tag removed.

**Comment by morleegan** [comment]
@eawest2 - seeing this error in the console. When on the `done` taskoptions.  

screen shot 2019-01-11 at 8 55 45 am

**Comment by eawest2** [comment]
Error corrected.

**Comment by eawest2** [comment]
I'm closing this PR and making a new branch. Morgan caught that I had apparently accidentally opened this branch from her changes regarding the weighted box.

**Pull request closed without merging**